catboost/libs/metrics/metric.h

```diff
@@ -1102,6 +1102,24 @@ struct TNormalizedGINI: public TNonAdditiveMetric {
     double Border = GetDefaultClassificationBorder();
 };
 
+//FairLoss metric
+struct TFairLossMetric: public TAdditiveMetric<TFairLossMetric> {
+    explicit TFairLossMetric(double c = 2);
+    TMetricHolder EvalSingleThread(
```

Comment:
Please add md file that explains changes that will be added to documentation here:
https://tech.yandex.com/catboost/doc/dg/concepts/loss-functions-docpage/

Reply:
Anton, I can help with English in the documentation :)